sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileCopyOptions.cs

```diff
@@ -64,5 +64,22 @@ public class ShareFileCopyOptions
         /// SMB properties to copy from the source file.
         /// </summary>
         public CopyableFileSmbProperties SmbPropertiesToCopy { get; set; }
+
+        /// <summary>
+        /// Only applicable to NFS Files.  NFS properties to set on the destination file.
+        /// </summary>
+        public FileNfsProperties NfsProperties { get; set; }
+
+        /// <summary>
+        /// Only applicable to NFS Files.  Specifies if the the mode should be copied from the sourece
+        /// to destination file, or overwritten with a new value for the destination.
+        /// </summary>
+        public ModeCopyMode? ModeCopyMode { get; set; }
+
+        /// <summary>
+        /// Only applicable to NFS Files.  Specifies if the the owner should be copied from the sourece
+        /// to destination file, or overwritten with a new value for the destination.
+        /// </summary>
+        public OwnerCopyMode? OwnerCopyMode { get; set; }
```

Comment:
TODO depending on what the Files team says, we may need to change this XML comment to say this parameter is mandatory for NFS files.

Reply:
Fixed.